applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_plane_strain_2D_law.h

```diff
@@ -95,63 +60,25 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoLinearElasticPlaneStrain2DLaw
      * @brief Dimension of the law:
      * @return The dimension were the law is working
      */
-    SizeType WorkingSpaceDimension() override
-    {
-        return Dimension;
-    }
+    SizeType WorkingSpaceDimension() override;
 
     /**
      * @brief Voigt tensor size:
      * @return The size of the strain vector in Voigt notation
      */
-    SizeType GetStrainSize() const override
-    {
-        return VoigtSize;
-    }
+    SizeType GetStrainSize() const override;
 
-    ///@}
-    ///@name Access
-    ///@{
-
-    ///@}
-    ///@name Inquiry
-    ///@{
-
-    ///@}
-    ///@name Input and output
-    ///@{
-
-    ///@}
-    ///@name Friends
-    ///@{
+    bool IsIncremental() override;
 
     /**
-     * @brief  Itreturns the value of a specified variable
+     * @brief  It returns the value of a specified variable
      * @param rThisVariable the variable to be returned
      * @param rValue a reference to the returned value
      * @param rValue output: the value of the specified variable
      */
     bool& GetValue(const Variable<bool>& rThisVariable, bool& rValue) override;
```

Comment:
To avoid hiding of member functions inherited from the `ConstitutiveLaw` class, I would add the following statement (to make them visible):
```
using ConstitutiveLaw::GetValue;
```
(This will resolve a problem found by SonarQube as well as by clang-tidy.)

Reply:
done